apps/web/tsconfig.json

```diff
@@ -3,12 +3,28 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "~/*": ["modules/*"],
-      "@components/*": ["components/*"],
-      "@lib/*": ["lib/*"],
-      "@server/*": ["server/*"],
-      "@prisma/client/*": ["@calcom/prisma/client/*"]
-    }
+      "~/*": [
+        "modules/*"
+      ],
+      "@components/*": [
+        "components/*"
+      ],
+      "@lib/*": [
+        "lib/*"
+      ],
+      "@server/*": [
+        "server/*"
+      ],
+      "@prisma/client/*": [
+        "@calcom/prisma/client/*"
+      ]
+    },
+    "plugins": [
+      {
+        "name": "next"
+      }
```

Comment:
Why was this necessary?

Reply:
As mentioned in the comment - this is added whenever i run yarn dev on main. 

next says its adding a plugin due to reccomendations.